src/components/accessories/billTable/BillTable.tsx

```diff
@@ -245,26 +235,51 @@ export const BillTable: FC<IBillTableProps> = ({
                     onChange={dateFieldHandleOnChange("toDate")}
                   />
                 </div>
-              </div>
-              <div className="row start-sm center-xs">
-                <div className="fullWidth filterBillForm__item">
-                  <PatientAutocomplete
+                <div className="filterBillForm__item">
+                  <DateField
+                    fieldName="month"
+                    views={["month"]}
+                    fieldValue={formik.values.month}
+                    disableFuture={false}
+                    theme="regular"
+                    format="dd/MM/yyyy"
+                    isValid={isValid("month")}
+                    errorText={getErrorText("month")}
+                    label={t("bill.month")}
+                    onChange={dateFieldHandleOnChange("month")}
+                  />
+                </div>
+                <div className="filterBillForm__item">
+                  <DateField
+                    fieldName="year"
+                    views={["year"]}
+                    fieldValue={formik.values.year}
+                    disableFuture={false}
+                    theme="regular"
+                    format="dd/MM/yyyy"
+                    isValid={isValid("year")}
+                    errorText={getErrorText("year")}
+                    label={t("bill.year")}
+                    onChange={dateFieldHandleOnChange("year")}
+                  />
+                </div>
+                <div className="filterBillForm__item">
+                  <PatientPicker
```

Comment:
It seems like that the components doesn't work as expected, when you try to digit something nothing appears.

Reply:
This is the patient picker that I design before. You should click on the loop to open search dialog.

Reply:
It is just a suggestion. I designed a patientAutocomplete and a patient Picker, but here I use the  PatientPicker.

Reply:
I think that the PatientAutocomplete is enough in this case.

Plus, the PatientPicker has some UI / UX bugs that I would like to discuss during the next call.